features/karaf/src/main/feature/feature.xml

```diff
@@ -55,7 +55,6 @@
     <bundle>mvn:org.eclipse.smarthome.io/org.eclipse.smarthome.io.monitor/${project.version}</bundle>
     <bundle>mvn:org.eclipse.smarthome.io/org.eclipse.smarthome.io.net/${project.version}</bundle>
     <bundle>mvn:org.eclipse.smarthome.io/org.eclipse.smarthome.io.rest/${project.version}</bundle>
-    <bundle>mvn:org.eclipse.smarthome.io/org.eclipse.smarthome.io.rest.auth/${project.version}</bundle>
```

Comment:
I wouldn't remove this bundle here - it is a part of the ESH base feature and it does not harm. With the same logic, you could otherwise also separate core.id, core.voice, io.console, io.rest.sse etc.